modules/systemd.nix

```diff
@@ -191,7 +213,7 @@ in
           (buildServices "timer" cfg.timers)
           ++
           (buildServices "path" cfg.paths)
-        );
+        )) // sessionVariables;
```

Comment:
This should not use `//` but `mkMerge`. I.e., something like
```
xdg.configFile = mkMerge [
  (listToAttrs (
    (buildServices "service" cfg.services)
    ++
    (buildServices "socket" cfg.sockets)
    ++
    (buildServices "target" cfg.targets)
    ++
    (buildServices "timer" cfg.timers)
    ++
    (buildServices "path" cfg.paths)
  ))

  sessionVariables
];
```

With `//` and having no session variables specified you get the unfortunate side-effect that all services, sockets, etc. are disabled as well. This is because of the way `mkIf` is defined, the expression ends up being a member of the attribute set that `//` will happily merge.

Suffice to say disabling everybodys services would be bad™ 😀

Reply:
Yeah, sorry. I failed to notice that my units disappeared :grimacing: